src/components/controls/MediaStreamControl.tsx

```diff
@@ -1,7 +1,19 @@
 import * as React from 'react';
 
+export interface IMediaStreamControlHandle {
```

Comment:
IMediaStreamControlRendererProps

Reply:
I want to understand the 4 methods usage.

Can we the a customize JSXElement which take this 4 as props?

Reply:
The `IMediaStreamControlRendererProps ` is 
````tsx
export interface ICustomMediaStreamControlProps {
    handle: IMediaStreamControlHandle;
}
```` 
I think it's some different with handle and props

Reply:
```tsx
export interface IMediaStreamControlRenderer {
    (handle: IMediaStreamControlHandle): JSX.Element | null | false;
}
```
should be =>

```tsx
export interface IMediaStreamControlRenderer {
    (props: IMediaStreamControlRendererProps): JSX.Element | null | false;
}
```

Reply:
Done. `this.props.props` is weird, so I destructed the props
````tsx
    private renderMediaStreamControl(props: IMediaStreamControlProps) {
        return <CustomMediaStreamControl {...props} />
    }
````